docs/development.md

```diff
@@ -36,3 +36,25 @@ build -c opt --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/object_
 'bazel-bin/mediapipe/examples/desktop/object_detection/object_detection_openvino --calculator_graph_config_file mediapipe/graphs/object_detection/object_detection_desktop_openvino_graph.pbtxt --input_side_packets "input_video_path=/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4,output_video_path=/mediapipe/tested_video.mp4"'
 ```
 - in object_detection example the input video will get all the objects detected by the model and return the tested_video.mp4 output.
+
+# Setting log level
+
+Openvino Model Server has its own log_level setting that can be set to the following levels:
+    INFO, DEBUG, WARNING, TRACE, ERROR
+
+Recommended development log_level setting is DEBUG as it will print out usefull information and perform additional OpenVINOInferenceCalculator calculator validation.
+If you want to set the flag in the desktop examples applications you can do it by setting the GLOG_minloglevel environment variable.
+If the GLOG_minloglevel variable is not set Model Server will use the default log_level - INFO.
+
+| GLOG_minloglevel | Model Server log_level |
+:------------------| :--------------------: |
+|                  |          INFO          |
```

Comment:
why don't you use the order from highest to lowest level - TRACE, DEBUG,INFO,WARNING, ERROR ?